### DISCUSSION TITLE
Upload attachments

### DISCUSSION THREAD

**Discussion by LiHongYan666** [requarks/wiki#6182] [Error / Bug Report]
I have two questions:
Problem 1: The maximum configuration of uploaded attachments can be set to 2 GB, but an error will be reported when uploading attachments exceeding 1 GB, and only within 1 GB can be uploaded and downloaded normally
Question 2: When I perform database PostgreSQL backup and restore, about 200MB of attachments can be restored normally, more than 200MB can not be restored normally, click the attachment to download will not report an error, but it has not been loaded, the source code will report warn

**Comment by LiHongYan666** [comment]
Problem 1: 
![图片](https://user-images.githubusercontent.com/125847393/220025028-6202ba90-790f-4e3f-8915-ea1efc0d086e.png)
![图片](https://user-images.githubusercontent.com/125847393/220025106-057be3e4-4ff8-4b26-8200-4c9f8a4955b3.png)
Question 2: 
![图片](https://user-images.githubusercontent.com/125847393/220025447-fcc4e5b0-18eb-42a5-8a54-186236e8c851.png)
![图片](https://user-images.githubusercontent.com/125847393/220025845-8de001af-c293-4b83-8f7e-5539edd939be.png)

**Comment by NGPixel** [comment]
Are you using a reverse proxy (nginx, cloudflare, apache, etc.) in front of Wiki.js?

You shouldn't store files this large in the wiki. Consider using a proper blob store like AWS S3, Azure Blob Storage, etc. for large files. Uploads within wiki.js are meant for images and documents, not 1-2GB files...

**Reply by LiHongYan666** [reply]
I didn't use proxy, now consider storing large files in other ways
suggest that question two can be optimized

**Reply by LiHongYan666** [reply]
Thanks for the suggestion
